### PR TITLE
[one-cmds] Enable change_outputs option

### PR DESCRIPTION
**Pull request by seanshpark**
This will enable change_outputs option in one-optimize.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6974, from draft #6976

**Review by mhs4670go — DISMISSED** [review]
LGTM

File: compiler/one-cmds/tests/one-optimize_002.test [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
```
Comment (seanshpark) [inline compiler/one-cmds/tests/one-optimize_002.test:3]: Ah, I need to change this -_-;

**Comment by seanshpark** [comment]
@mhs4670go , PTAL again -_-;;;

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**